braid/twisted/service.py

```diff
@@ -54,10 +54,12 @@ def bootstrap(self, python='pypy'):
             pip.install('twisted', python=python)
 
             # Create base directory setup
-            run('/bin/mkdir -p {} {} {}'.format(
+            run('/bin/mkdir -p {} {} {} {}'.format(
                 self.runDir,
                 self.logDir,
-                self.binDir))
+                self.binDir,
+                self.configDir,
+                ))
```

Comment:
there are 5 lights!@#!@#!@#

(OK, I have no idea what I was thinking, you're right.)